spec/_attachments/ic.did

```diff
@@ -2,244 +2,305 @@ type canister_id = principal;
 type wasm_module = blob;
 
 type canister_settings = record {
-  controllers : opt vec principal;
-  compute_allocation : opt nat;
-  memory_allocation : opt nat;
-  freezing_threshold : opt nat;
-  reserved_cycles_limit : opt nat;
+	controllers : opt vec principal;
```

Comment:
Could we please stick with spaces instead of tabs (e.g., with 4 spaces instead of 2 spaces, but avoiding tabs)?